proto/minder/v1/minder.proto

```diff
@@ -831,13 +831,14 @@ message CreateUserRequest { }
 
 message CreateUserResponse {
     int32 id = 1;
-    string organization_id = 2;
-    string organizatio_name = 3;
     string project_id = 4;
     string project_name = 5;
     string identity_subject = 6;
     google.protobuf.Timestamp created_at = 7;
     Context context = 8;
+
+    reserved 2, 3; // deprecated organization_id, organization_name
+    reserved "organization_id", "organizatio_name";
```

Comment:
I meant the following:

```suggestion
    string organization_id = 2 [deprecated=true]
    string organization_name =3 [deprecated=true]
```

Which will warn us when we use those fields, but will still allow us to set them for older clients.